src/main/java/com/revature/responsecollector/JMeterResponseCollector.java

```diff
@@ -0,0 +1,178 @@
+package com.revature.responsecollector;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.jmeter.reporters.ResultCollector;
+import org.apache.jmeter.reporters.Summariser;
+import org.apache.jmeter.samplers.SampleEvent;
+import org.apache.jmeter.samplers.SampleResult;
+
+public class JMeterResponseCollector extends ResultCollector { 
+    
+    private static final long serialVersionUID = 1L;
```

Comment:
If this field is to be included, it should have a generated value instead of a default value.